packages/engine/src/debug/systems/DebugHelpersSystem.ts

```diff
@@ -47,56 +42,19 @@ const quat = new Quaternion()
 const cubeGeometry = new ConeBufferGeometry(0.05, 0.25, 4)
 cubeGeometry.rotateX(-Math.PI * 0.5)
 
-export default async function DebugHelpersSystem(world: World) {
-  const helpersByEntity = {
-    viewVector: new Map(),
-    ikExtents: new Map(),
-    box: new Map<Entity, Box3Helper>(),
-    helperArrow: new Map(),
-    velocityArrow: new Map(),
-    navmesh: new Map(),
-    navpath: new Map()
-  }
+export const helpersByEntity = {
```

Comment:
we want to keep this in the system scope rather than global scope such that we can write tests more easily. we want to avoid mutable global scope where possible.

Reply:
We can freeze the object or we have to revert back to previously implemented logic where receptors are in the `DebugHelperSystem`

Reply:
oh i see, yes let's move the receptors back and keep it scoped to the system